locations/philmel.yml

```diff
@@ -0,0 +1,198 @@
+
+  - hostname: philmel-ost
+    address: 10.230.2.7
+    snmp_profile: airos_6
+
+  - hostname: philmel-vaterhaus
+    address: 10.230.2.8
+    snmp_profile: airos_8
+
+  - hostname: philmel-sued
+    address: 10.230.2.9
+    snmp_profile: airos_6
+
+  - hostname: philmel-sued-5ac
+    address: 10.230.2.10
+    snmp_profile: airos_8
+
+  - hostname: philmel-west
+    address: 10.230.2.11
+    snmp_profile: airos_6
+
+  - hostname: philmel-ak36
+    address: 10.230.2.12
+    snmp_profile: airos_8
+
+# TODO: Add all devices and
+# change naming
+airos_dfs_reset:
+  - name: "philmel-ak36"
+    target: "10.230.2.12"
+    username: "ubnt"
+    password: "file:/root/pwd.txt"
+    daytime_limit: "2-7"
+  - name: "philmel-nord-5ac"
+    target: "10.230.2.6"
+    username: "ubnt"
+    password: "file:/root/pwd.txt"
+    daytime_limit: "2-7"
+
+ipv6_prefix: "2001:bf7:820:1500::/56"
+
+# Complete subnet: 10.230.2.0/24
+# MGMT Network: 10.230.2.0/28
+# Mesh Network: 10.230.2.16/28 (-23)
+# DHCP Network: 10.230.2.32/28
+
+networks:
+  - vid: 2
+    role: dhcp
+    prefix: 10.230.2.32/28
+    ipv6_subprefix: 0
+    untagged: true
+    inbound_filtering: true
+    enforce_client_isolation: true
+    assignments:
+      philmel-core: 1
+  # northeast mesh 5GHz ac
+  - vid: 10
+    role: mesh
+    name: mesh_no_5ghz         # Peers: kiehlufer-core
+    prefix: 10.230.2.17/32
+    ipv6_subprefix: -1
+    mesh_metric: 1024
+  # northwest mesh 5GHz ac
+  - vid: 11
+    role: mesh
+    name: mesh_nw_5ghz          # Peers: liegewiese, emser97
+    prefix: 10.230.2.18/32
+    ipv6_subprefix: -2
+    mesh_metric: 1024
+  - vid: 12
+    role: mesh
+    name: mesh_ost            # Peers: Area51, delbrueck66
+    prefix: 10.230.2.19/32
+    ipv6_subprefix: -3
+    mesh_metric: 1024
+  - vid: 13
+    role: mesh
+    name: mesh_nf_east
+    prefix: 10.230.2.20/32
+    ipv6_subprefix: -4
+    mesh_metric: 1024
+    mesh_ap: philmel-nf-east-5ghz
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+  - vid: 14
+    role: mesh
+    name: mesh_sued           # Peers: kranold18, GSBS2
+    prefix: 10.230.2.21/32
+    ipv6_subprefix: -5
+    mesh_metric: 1024
+  - vid: 15
+    role: mesh
+    name: mesh_sued_5ghz
+    prefix: 10.230.2.22/32
+    ipv6_subprefix: -6
+    mesh_metric: 1024
+  - vid: 16
+    role: mesh
+    name: mesh_west
+    prefix: 10.230.2.23/32
+    ipv6_subprefix: -7
+    mesh_metric: 1024
+  - vid: 17
+    role: mesh
+    name: mesh_ak36
+    prefix: 10.230.2.24/32
+    ipv6_subprefix: -8
+    ptp: true
+    mesh_metric: 1024
+    mesh_metric_lqm: ['default 0.2']  # prefer klunker link
+  - vid: 18
+    role: mesh
+    name: mesh_klunker
+    prefix: 10.230.2.25/32
+    ipv6_subprefix: -9
+    ptp: true
+    mesh_metric: 128
+  - vid: 19
+    role: mesh
+    name: mesh_nw_60ghz  # peers: sgfrd-core
+    prefix: 10.230.2.26/32
+    ipv6_subprefix: -10
+    ptp: true
+    mesh_metric: 1024
+  - vid: 20
+    role: mesh
+    name: mesh_nf_south
```

Comment:
This name is too long. The maximum length of an interface name in linux is 15 characters and this will be prefixed with `br-` and therefore it has to be shortened so this AP will build. Note: If you build the error message states exactly this. On other locations we use the following naming convention here: `mesh_11s_[direction]_[band]` e.g. `mesh_11s_s5`